src/components/modals/resetpassword_modal.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import InputField from "../fields/input_field";
+
+interface ResetPasswordModalProps {
+    isOpen: boolean;
+    onClose: () => void;
+    email: string;
+    onEmailChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+    onReset: () => void;
+    message: string;
+    isError: boolean;
+    isSending: boolean;
+}
+
+const ResetPasswordModalProps: React.FC<ResetPasswordModalProps> = ({
```

Comment:
Remove props from component name
```suggestion
const ResetPasswordModal: React.FC<ResetPasswordModalProps> = ({
```